gammapy/modeling/models/prior.py

```diff
@@ -171,12 +176,12 @@ class UniformPrior(Prior):
 
     Parameters
     ----------
-    min : float
+    min : float, optional
         Minimum value.
-        Default is -inf.
-    max : float
+        Default is -`np.inf`.
```

Comment:
```suggestion
        Default is -`numpy.inf`.
```

By changing these to 'numpy' we actually have a proper link in the docs :partying_face: